public/js/index.js

```diff
@@ -1,70 +0,0 @@
-var table = document.querySelector('#weeks-mentor-table');
-var cohorts = document.querySelector('.cohort-form-input');
-var viewButton = document.querySelector('#view');
-var addButton = document.querySelector('#add');
-var cohortForm = document.querySelector('.addcohort-container');
-var tableDiv = document.querySelector('.table-container');
-var addMentorButton = document.querySelector('#add-mentor');
-var mentorAccount = document.querySelector('#account')
-var weekNumber = document.querySelector('#week-number');
-
-
-get('/cohorts', (response) => {
-  document.getElementById("CohortName").innerHTML = generateOptions(response);
-})
-
-
-function generateOptions(e) {
-  let strt = "<option value=''>Select Your Cohort</option>";
-  return e.reduce(function(ss, op) {
-    return strt += "<option id='" + op.id + "' 'value='" + op.name + "'>" + op.name + "</option>"
-  }, "");
-}
-
-viewButton.onclick = (e) => {
-  var selectedCohort = cohorts.options[cohorts.selectedIndex].value;
-  var selectedID = cohorts.options[cohorts.selectedIndex].id;
-
-  var url = `weeks?cohort=${selectedID}`;
-
-  get(url, (response) => {
-    var weekNumbers = Object.keys(response);
-    var rows = table.getElementsByTagName('tr');
-    var rowsArray = Array.from(rows);
-    if (rowsArray.length > 0) {
-      rowsArray.forEach((row, i) => {
-        table.deleteRow(row);
-      });
-    }
```

Comment:
The if isn't necessary, you can use `forEach` on an empty array.

Nice use of `Array.from`.